.github/pull_request_template.md

```diff
@@ -1,80 +1,35 @@
+Main description of what's done here, shortly, with reasoning, references, thoughts and ideas for better understanding.
+This text is intended to be removed and replaced with actual description. Other items can also be removed or edited if not relevant. If any of following items require more description, it should also go in this block.
+Please, read all items and make an action every time to create more conscious PR. Thanks!
+
 ### PR fulfills these requirements
-- [ ] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
-- [ ] Tests for the changes have been added/updated (for bug fixes/features)
-- [ ] Docs have been added/updated (for bug fixes/features)
+- [ ] Tests for the changes have been added/updated
+- [ ] Docs have been added/updated
 - [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
-- [ ] Self-reviewed and ran all changes on a local instance (for bug fixes/features)
-
-
-
-#### Change has impacts in these area(s)
-_(check all that apply)_
-- [ ] Product design
-- [ ] Frontend
-
-
+- [ ] Self-reviewed and ran all changes on a local instance
 
 ### Describe the reason for change
-_(link to issue, supportive screenshots etc.)_
-
-
-
-#### What does this fix?
-_(if this is a bug fix)_
-
-
-
-#### What is the new behavior?
-_(if this is a breaking or feature change)_
-
-
-
-#### What is the current behavior?
-_(if this is a breaking or feature change)_
-
-
+(if it's not clear from description)
 
-#### What libraries were added/updated?
-_(list all with version changes)_
-
-
-
-#### Does this change affect performance?
-_(if so describe the impacts positive or negative)_
-
-
-
-#### Does this change affect security?
-_(if so describe the impacts positive or negative)_
-
-
-
-#### What alternative approaches were there?
-_(briefly list any if applicable)_
-
-
-
-#### What feature flags were used to cover this change?
-_(briefly list any if applicable)_
+### What feature flags were used to cover this change?
 
+### What alternative approaches were there?
```

Comment:
```suggestion
```

I think this information would be best kept in a TDD or another document. As we had discussed the heft of this template, it had been also mentioned this was duplicating that effort in the best case, worst case was just too much noise for the day to day work.

Reply:
One of the main reasons to read old PRs is to understand what and why was done. And this section will help to understand why.